decidim-core/app/views/decidim/shared/_resource_kebab.html.erb

```diff
@@ -0,0 +1,18 @@
+<div class="relative ml-auto">
+  <button id="dropdown-trigger-resource-<%= resource.id %>"
+          data-component="dropdown"
+          data-target="dropdown-menu-resource-<%= resource.id %>"
+          aria-label="<%= t("decidim.resource.controls_label") %>">
+    &#8942;
```

Comment:
This should be a "more-2" icon in remix icon as far as I understand, to be consistent with other icons

Reply:
I guess that we need to do the same thing in comments : 
https://github.com/decidim/decidim/blob/802ee6a2434f8580a87c55a6956748a092cec4c1/decidim-comments/app/cells/decidim/comments/comment/show.erb#L23-L25

Reply:
Yes, let's do that too please

Reply:
I will do in a separate PR